### ISSUE TITLE
more like summon bullshit

### ISSUE THREAD

**Issue by ShadowDeath6** [yogstation13/yogstation#656]
some magic shit you get is either grief or literally round ruining

horsehead- this doesn't do jack shit to wizards, it's just used to be a dumb asshole to your crewmates, don't even lie

staff of change / wand of polymorph- you get hit by this you're permafucked because you can't get cloned if you get griefed, and if you're turned into a xeno you're going to get killed by some dumb asshole, and if you roll drone or something you either beat the shit out of the guy who transformed you and turn yourself back or get bwoinked because admin discrimination says polymorphed drones can / can't retaliate against the dumb griefers who attacked them

it gives the wizard a spell point for making the crew grief each other

INB4 SOMEONE SAYS WIZ CAN GET FUCKED BY MAGIC

RISE! will counter literally all magic, including polymorph provided he doesn't roll something really really bad that deletes on death, I don't even know if that can happen

**Comment by timkoster1** [comment]
I am happy with a simple wicker doll.
>:)

**Comment by Super3222** [comment]
what are you even on about shadow.
wizards are meant to ruin peoples rounds. their tricks are suppose to be some what dumb, somewhat strategic, somewhat plain 'THAT'S FUCKING OVERPOWERED, PLS NERF'. that's the essence of the mode.

and the entire point of summon guns & summon magic was to attract griefers. a part of the wizards power is literally feeding them, don't lie. the only reason anyone would consider summon events would be if they're bored, want a "challenge", ragin magin, or want to attract griefers including another one of those choices.
that's also what's hilarious about those modes because it can turn back on the wizard real fast.

**Comment by ShadowDeath6** [comment]
summon magic gives other crewmembers the capability to grief each other stupidly easily and most of the stuff that they normally do to each other won't affect the wizard if he's half competent, it might as well always be picked because the wizard gets a free spellpoint and makes the crew kill each other for him

**Comment by Super3222** [comment]
I thought it didn't give wizards a free spell point anymore... if it does, it shouldn't really.

As for guns, that's another road, that's actually pretty split. I don't see anything wrong with a wizard getting one little point after it, and especially with the spellbook refactor.

**Comment by timkoster1** [comment]
Wizard usually gets hit by a bolt of death, and ya wouldn't want the admins to get bored.

**Comment by ShadowDeath6** [comment]
>wizard is hit by a bolt of death
>immediately RISE!s

back in black

**Comment by AdamElTablawy** [comment]
@timkoster1 

that's a stupid thing to say

@Super3222 
it is not supposed to attract griefers. that makes no sense. a mechanic that attracts griefers makes absolutely no sense. we have rules against griefing. all that does is ruin the round for people and give admins more work to do banning anyone who uses magic harmfully on other crew.

if you want to fix this, make summon magic make people either get 'steal 5 magical artifacts and survive till the end of the round!' as an objective like summon guns does, or make summon magic give random people to become the wizard's apprentice by dueling with other magical apprentices, and they win if they're the last person with the objective left alive at round end. meaning they have to kill all other apprentices, but not kill the wizard, or the crew.

**Comment by Super3222** [comment]
right, let me rephrase myself. one of the reasons to consider using it is to get the crew can unintentionally/intentionally grief themselves with magic. it's a distraction which sometimes makes the station devolve into choas. Like how it helps the wizard when some idiot is using magic in self defense or some random assistant is running around barnyarding everyone. but like summon guns has a backlash against the wiz, less drastic, but hey

apprentice DM is not bad, as long as you can see who's an apprentice with a HUD icon and who's not. :+1:

**Comment by Kmc2000** [comment]
The point of summon magic IS to get them to grief themselves to autocomplete your objectives or make a distraction


Even read the description for the spell and it's apparent

**Comment by ShadowDeath6** [comment]
free spellpoint for free greentext?
